autoload/neoterm/repl.vim

```diff
@@ -24,15 +24,25 @@ function! neoterm#repl#term(id)
   if has_key(g:neoterm.instances, a:id)
     let g:neoterm.repl.instance_id = a:id
     let g:neoterm.repl.loaded = 1
-
+    let l:cmd = [g:neoterm_repl_command, g:neoterm_eof]
+    if g:neoterm_repl_python_venv != ''
+      call insert(l:cmd, g:neoterm_repl_python_venv, 0)
+    end
     if !empty(get(g:, 'neoterm_repl_command', ''))
           \ && g:neoterm_auto_repl_cmd
           \ && !g:neoterm_direct_open_repl
       call neoterm#exec({
-            \ 'cmd': [g:neoterm_repl_command, g:neoterm_eof],
+            \ 'cmd': l:cmd,
             \ 'target': g:neoterm.repl.instance().id
             \ })
     end
+    if g:neoterm_clean_startup == 1
```

Comment:
Ok, i understand your opinion. I can handle this in my personal settings.
It would only benefit if the screen update was interrupted during initialization.